modules/vpn_ha/main.tf

```diff
@@ -60,10 +61,17 @@ resource "google_compute_external_vpn_gateway" "external_gateway" {
   }
 }
 
+data "google_compute_router" "router" {
+  name    = var.router_name
```

Comment:
If someone sets `router_name` null (as shown in the code below) module will use null instead of default value empty string `""`. In this case data will fail as `name` is required variable and code will throw an error. Do you think we should add a logic here `var.router_name == null ? "" : var.router_name` in order to avoid error?

```
module my_vpn {
source = ""
router_name = null
....
}
```

Reply:
Valid point. I've added a check.